backend_server/configs/oauth.yaml

```
@@ -0,0 +1,12 @@
+google:
```

probably better to rename this file to `oauth.DEFAULT.yaml` or something so that it fails hard if you don't configure it before running.  Also keeps the secret keys absolutely distinct from the versioned files.

Also add the real config file path (whatever it is) to the top-level .gitignore.